cli/src/cli_util.rs

```diff
@@ -413,6 +418,40 @@ impl CommandHelper {
             })
     }
 
+    pub fn load_stale_working_copy_commit(
```

Comment:
I don't think we need to rewrite `for_stale_working_copy()` not to return `WorkspaceCommandHelper` in this patch. If it returned `WorkspaceCommandHelper`, the caller can just reuse it in the next patch, I think.

If `Recovered`, the `workspace_command` is instantiated for the current op, so it can be reused. If `Snapshotted`, the caller will create new `workspace_command` anyway, so it can be done in this function.